dali/pipeline/operators/arithmetic/arithmetic_meta.h

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_PIPELINE_OPERATORS_ARITHMETIC_ARITHMETIC_META_H_
+#define DALI_PIPELINE_OPERATORS_ARITHMETIC_ARITHMETIC_META_H_
+
+#include <cstdint>
+#include <type_traits>
+
+#include "dali/core/cuda_utils.h"
+#include "dali/core/static_switch.h"
+#include "dali/pipeline/data/types.h"
+
+namespace dali {
+
+enum class ArithmeticOp {
+  id,
```

Comment:
What is the `id` operation? Please add a comment (rest are obvious, this one is not)

Reply:
Changed to `plus` and `minus` - unary arithmetic operators.